bugalink-backend/users/models.py

```diff
@@ -1,6 +1,20 @@
 from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.core.files.storage import FileSystemStorage
 from django.db import models
 
+def user_directory_path(instance, filename):
+    # print(instance.user)
+    return "users/{0}/{1}".format(instance.id, filename)
+
+class OverwriteStorage(FileSystemStorage):
```

Comment:
You have fixed the problem of redundant files (avatar.png, avatar.jpeg) in `users/models.py` but it's still occuring in the `class OverwriteStorage()` in `driver/models.py`. Fix it there too if possible

Reply:
Will do